run_route_scripts/gen_multipoint_route_requests.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
```

Comment:
to me its sad to duplicate the code in the other script with some changes. i worry the more we do this, the more code we have that is unmaintained. is there a way to add this functionality to the existing script?

Reply:
looks like you did combine this into the other script now. can we delete this one?